src/components/structures/RoomView.js

```diff
@@ -91,6 +90,7 @@ module.exports = React.createClass({
         return {
             room: null,
             roomId: null,
+            userId: null,
```

Comment:
Looks like we don't use this anywhere? We yank it from `MatrixClientPeg.get().credentials.userId` on `:1619`.